src/main/kotlin/com/milaboratory/mixcr/alleles/CloneRebuild.kt

```diff
@@ -55,7 +55,7 @@ class CloneRebuild(
     assemblingFeatures: GeneFeatures,
     private val threads: Int,
     cloneFactoryParameters: CloneFactoryParameters,
-    featuresToAlign: Map<GeneType, GeneFeature>
```

Comment:
?

Reply:
I don't know when and where we start to write null in maps like this. May be somewhere there is a circle on each gene type

Reply:
Ok, got it.